AWSPinpoint/AWSPinpointNotificationManager.m

```diff
@@ -91,16 +93,15 @@ - (BOOL)interceptDidFinishLaunchingWithOptions:(nullable NSDictionary *)launchOp
     NSDictionary *notificationPayload = launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
     if(notificationPayload)
     {
-        if (![AWSPinpointNotificationManager validCampaignPushForNotification:notificationPayload]) {
+        if (![AWSPinpointNotificationManager validPinpointPushForNotification:notificationPayload]) {
             return YES;
         }
-        
-        [self addGlobalCampaignMetadataForNotification:notificationPayload];
-        
+        NSDictionary *metadata = [self getMetadataFromUserInfo:notificationPayload];
+        [self addGlobalPinpointMetadata:metadata];
+
         // Application launch because of notification
         [self recordMessageOpenedEventForNotification:notificationPayload
-                                       withIdentifier:nil
-                                 withApplicationState:[[UIApplication sharedApplication] applicationState]];
```

Comment:
This function was previously calling:
```
   [self addApplicationStateAttributeToEvent:pushNotificationEvent	
                         withApplicationState:state];
```

Doesn't seem that we are doing this anymore?

To be honest, I'm having a lot of trouble trying to figure out of this code still works...  There were quite a bit of changes, I'm not familiar with the code in the first place, hard to know if we caused any regression.

Do you have a way of sending out smaller CRs so that it is easier to see a progression of changes (for this file only -- the other files, it's obvious what is being changed)?

Reply:
This attribute is added to all push notification events, so I moved it to `buildEventFromUserInfo:withPushActionType` Let's discuss what's the best way to show the progression